### PR TITLE
Action Based Account setup changes

### PR DESCRIPTION
**Pull request by shabeeb-aot**
*Issue #:*
https://github.com/bcgov/entity/issues/7533

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

File: auth-web/src/components/auth/common/Product.vue [hunk 1]
```diff
@@ -102,131 +85,79 @@ export default class Product extends Vue {
...
-  public icon = ''
-  public isApproved = false
-  public isPending = false
-  public isRejected = false
-  public isRequesting = false
-  public isHighlighted = false
+  public abcd:boolean = false
 
+  public productSelected:boolean = false
+public newproductSelected = 'tes'
```
Comment (mengdong19) [inline auth-web/src/components/auth/common/Product.vue:96]: space?

File: auth-web/src/components/auth/common/Product.vue [hunk 2]
```diff
@@ -5,70 +5,59 @@
...
             <div class="pr-8 ">
-              <h3 class="title font-weight-bold product-title mt-n1">{{productDetails.name}}</h3>
-              <div>{{productSubTitle}}</div>
+              <v-checkbox
+                color="primary"
+                class="align-checkbox-label--top ma-0 pa-0"
+                hide-details
+                v-model="productSelected"
+                :indeterminate="isexpandedView && isTOSNeeded && !termsAccepted"
+                :data-test="`check-product-${productDetails.description}`"
```
Comment (sumesh-aot) [inline auth-web/src/components/auth/common/Product.vue:20]: data-test can be code instead of description, as description contains spaces

**Review by sumesh-aot — APPROVED** [review]

File: auth-web/src/components/auth/account-settings/product/ProductPackage.vue [hunk 3]
```diff
@@ -108,34 +114,45 @@ export default class ProductPackage extends Mixins(AccountChangeMixin) {
...
+        // this.loadProduct()
         this.productsAddSuccess = true
       } catch {
         // open when error
         this.$refs.errorDialog.open()
       }
     }
   }
+
+  public showAsSelected (productStatusCode) {
```
Comment (saravanpa-aot) [inline auth-web/src/components/auth/account-settings/product/ProductPackage.vue:128]: type productStatusCode and also should it be public?

File: auth-web/src/components/auth/common/Product.vue [hunk 4]
```diff
@@ -102,147 +83,83 @@ export default class Product extends Vue {
...
+  onisSelectedChange (newValue:boolean) {
+    // setting check box
+    this.productSelected = newValue
   }
 
-  get isDisableSaveBtn () {
-    return !this.isFormvalid()
+  get productLabel () {
+    const { code } = this.productDetails
+    const subTitle = `${code && code.toLowerCase()}CodeSubtitle` || ''
```
Comment (saravanpa-aot) [inline auth-web/src/components/auth/common/Product.vue:104]: please add more comments on this logic..

**Review by saravanpa-aot — APPROVED** [review]

**Pull request merged**